software/util/reorg.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
consider adding the emacs + encoding header.

#!/usr/bin/env python3
# -*- coding: UTF-8 -*-